.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -9,20 +9,20 @@ body:
   - type: markdown
     attributes:
       value: |
-        Before crreating a feature request, please search for the behaviour in the existing issues. Keep in mind that we use the [agile style](https://www.agilealliance.org/glossary/user-story-template/) for our user stories
+        Before creating a feature request, please search for the behaviour in the existing issues. Keep in mind that we use the [agile style](https://www.agilealliance.org/glossary/user-story-template/) for our user stories
         
         - The *Who* should be included in the title. E.g. `As a user I want ...`
-        - The *What* should also  be included in the title. E.g. `a good site map`
-        - The *Why* should be defined in the firt paragraph of the user story 
+        - The *What* should also be included in the title. E.g. `a good site map`
+        - The *Why* should be defined in the first paragraph of the user story 
         ---
         
         Thank you for taking the time to file a feature request. To procces this request as fast as possible, we need some information.
   - type: textarea
     id: why
     attributes:
-      placeholder: "Zodat ...."
+      placeholder: "So ...."
       label: What is the goal of your request? Why should we do this?
-      description: a clear and concise description of what the question is that you ar trying to solve. E.g.,  Zodat ik niet op 5 knopen moet drukken voordat ik antwoord heb  op mijn vraag
+      description: a clear and concise description of what the question is that you are trying to solve. E.g.,  So I don't have to click five times before reaction
```

Comment:
I find this example we're giving not so clear. Imagine someone would create a feature-request and the goal would be "So I don't have to click five times before reaction". 

And then, the acceptance criteria would be "So I can link from the homepage". 

Unsure how we would process that.

Reply:
I think with better wording, you don't need an example. (imho)